src/main/kotlin/lotto/domain/LottoPrizes.kt

```diff
@@ -0,0 +1,20 @@
+package lotto.domain
+
+enum class LottoPrizes(
+    val equalCount: Int,
+    val money: Int
+) {
+    MATCH_THREE_PRIZES(3, 5000),
+    MATCH_FOUR_PRIZES(4, 50000),
+    MATCH_FIVE_PRIZES(5, 1500000),
+    MATCH_SIX_PRIZES(6, 2000000000);
```

Comment:
```suggestion
    MATCH_THREE_PRIZES(3, 5_000),
    MATCH_FOUR_PRIZES(4, 50_000),
    MATCH_FIVE_PRIZES(5, 1_500_000),
    MATCH_SIX_PRIZES(6, 2_000_000_000);
```
언더 스코어를 활용하여 가독성을 높일수 있어요 😄

Reply:
언더 스코어를 추가했습니다